doc/source/User_guide/pyaedt_extensions_doc/project.rst

```diff
@@ -0,0 +1,49 @@
+Import Nastran
+==============
+
+Import Nastran or STL file in any 3D modeler application. The user can also preview the imported file and decimate it before import it.
+
+User can access the extension from the icon created in the Automation tab through the extension manager.
+The following image is the extension user interface:
+
+.. image:: import_nastran_ui.png
+  :width: 800
+  :alt: Import nastran UI
+
+User can launch the user interface from the terminal:
+
+.. code::
+
+   SET PYAEDT_SCRIPT_PORT=50051
+   SET PYAEDT_SCRIPT_VERSION=2024.1
+   python.exe path/to/pyaedt/workflows/project/import_nastran.py
+
+The available arguments are: file_path, planar, lightweight, and decimate. These arguments can be obtained from the help:
+
+.. code::
+
+   python.exe path/to/pyaedt/workflows/project/import_nastran.py --help
+
+For example, you can pass the input file as an argument, and in this case the user interface is not launched:
+
+.. code::
+
+   export PYAEDT_SCRIPT_PORT=50051
+   export PYAEDT_SCRIPT_VERSION=2024.1
+   python.exe path/to/pyaedt/workflows/project/import_nastran.py --file_path="my_file.stl"
+
+Fianlly, the extension can be used in a python script without the user interface with the following code:
```

Comment:
```suggestion
Finally, the extension can be used in a Python script from the command prompt with the following code:
```

Reply:
Rewrite in active voice: 
Finally, this code shows how you can run the extension directly from a Python script: